src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Dependencies/SinkDependenciesFactory.cs

```diff
@@ -56,10 +56,10 @@ internal static class SinkDependenciesFactory
                     ? (ISqlBulkBatchWriter)new SqlBulkBatchWriter(
                         sinkOptions.TableName, sinkOptions.SchemaName, columnOptions.DisableTriggers,
                         sqlConnectionFactory, logEventDataGenerator)
-                    : (ISqlBulkBatchWriter)new SqlInsertBatchWriter(
+                    : (ISqlBulkBatchWriter)new SqlInsertStatementWriter(
```

Comment:
Looks like it was required unless I upgraded the C# language version.  Didn't know if that would be totally safe or not.

![image](https://github.com/serilog-mssql/serilog-sinks-mssqlserver/assets/6400605/a562532d-1bef-41c8-b22b-98d165a66e2d)

Reply:
Correct, I've noticed that today and then resolved my comment. We're stuck on C# 7.3 since we have to support .NET Framework. So the cast is required. :)